crates/ruff_linter/src/rules/pyupgrade/rules/use_pep604_isinstance.rs

```diff
@@ -39,9 +39,13 @@ impl CallKind {
 ///
 /// ## Why is this bad?
 /// Since Python 3.10, `isinstance` and `issubclass` can be passed a
-/// `|`-separated union of types, which is more concise and consistent
+/// `|`-separated union of types, which is consistent
```

Comment:
I also changed this line, since the rule only makes code more concise if the tuple of types has exactly length 2. If the tuple is 3 elements long, the rule results in code that has exactly the same level of verbosity. If the tuple is 4 or more elements long, the rule results in code that is more verbose than it was originally.